packages/nextra-theme-docs/src/utils/getHeadingText.ts

```diff
@@ -1,9 +1,5 @@
 import { Heading } from 'nextra'
 
 export default function getHeadingText(heading: Heading) {
-  if (Array.isArray(heading.children) && heading.children.length === 1) {
-    const content = heading.children[0]
-    if (content.type === 'text') return content.value
-  }
-  return ''
+  return heading.value || ''
```

Comment:
This will make docs example like this

![plain-text.png](https://vip2.loli.io/2022/01/09/sOlB3WjAbNYdfvV.png)

I think we should only render headers which are plain-text in TOC ?

Reply:
Thanks, just fixed it!